e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -54,4 +54,35 @@ describe('Customization', () => {
             expect(config.TeamSettings.SiteName).to.eq(siteName);
         });
     });
+
+    it('SC20329 - Can change Terms of Service Link setting', () => {
+        // * Verify site name's setting name for is visible and matches the text
+        cy.findByTestId('SupportSettings.TermsOfServiceLinklabel').scrollIntoView().should('be.visible').and('have.text', 'Terms of Service Link:');
+
+        // * Verify the site name input box has default value. The default value depends on the setup before running the test.
+        cy.findByTestId('SupportSettings.TermsOfServiceLinkinput').should('have.value', origConfig.SupportSettings.TermsOfServiceLink);
+
+        // * Verify the site name's help text is visible and matches the text
+        cy.findByTestId('SupportSettings.TermsOfServiceLinkhelp-text').find('span').should('be.visible').and('have.text',
+            'Link to the terms under which users may use your online service. By default, this includes the ' +
+            '"Mattermost Conditions of Use (End Users)" explaining the terms under which Mattermost software is ' +
+            'provided to end users. If you change the default link to add your own terms for using the service you ' +
+            'provide, your new terms must include a link to the default terms so end users are aware of the Mattermost ' +
+            'Conditions of Use (End User) for Mattermost software.');
+
+        // # Generate and enter a random help link
+        const newValue = 'https://test.com';
+        cy.findByTestId('TeamSettings.SiteNameinput').clear().type(newValue);
```

Comment:
I think this should be
```suggestion
        cy.findByTestId('SupportSettings.TermsOfServiceLinkinput').clear().type(newValue);
```

Reply:
Such a stupid mistake :/. Done :+1: